testutil/base.go

```diff
@@ -34,6 +36,11 @@ func (s *BaseTest) SetUpTest(c *check.C) {
 	if len(s.cleanupHandlers) != 0 {
 		panic("BaseTest cleanup handlers were not consumed before a new test start, missing BaseTest.TearDownTest call?")
 	}
+	// Just to ensure SNAPD_DEBUG unset at the moment when called
```

Comment:
This is not a very useful comment, it just writes  down what the code already does. Instead it should describe why this is done/needed.